update README to include links to GPU processing, LLM's, and the spaCy blog. (#13197)
explosion/spaCy: README.md

| 🪐 **[Project Templates]**                                                                                                                                                                                                | End-to-end workflows you can clone, modify and run.                                                                                                                                                                                                                                                                                          |
| 🎛 **[API Reference]**                                                                                                                                                                                                     | The detailed reference for spaCy's API.                                                                                                                                                                                                                                                                                                      |

> | ⏩ **[GPU Processing]**                                                                                                                                                                                                    | Use spaCy with CUDA compatible GPU processing.                                                                                                                                                                                                                                                                                               |

Reviewer: ```suggestion
| ⏩ **[GPU Processing]**                                                                                                                                                                                                    | Use spaCy with CUDA-compatible GPU processing.                                                                                                                                                                                                                                                                                               |
```

| 📦 **[Models]**                                                                                                                                                                                                           | Download trained pipelines for spaCy.                                                                                                                                                                                                                                                                                                        |

> | 🦙 **[Large Language Models]**                                                                                                                                                                                            | Integrate LLM's into spaCy pipelines.                                                                                                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| 🦙 **[Large Language Models]**                                                                                                                                                                                            | Integrate LLMs into spaCy pipelines.                                                                                                                                                                                                                                                                                                        |
```

| ⚙️ **[spaCy VS Code Extension]**                                                                                                                                                                                          | Additional tooling and features for working with spaCy's config files.                                                                                                                                                                                                                                                                       |
| 👩‍🏫 **[Online Course]**                                                                                                                                                                                                    | Learn spaCy in this free and interactive online course.                                                                                                                                                                                                                                                                                      |

> | 📰 **[Blog]**                                                                                                                                                                                                             | Read foundational ideas that led to the formation of spaCy and current development.                                                                                                                                                                                                                                                          |

Reviewer: ```suggestion
| 📰 **[Blog]**                                                                                                                                                                                                             | Read about current spaCy and Prodigy development, releases, talks and more from Explosion.                                                                                                                                                                                                                 |
```

[new in v3.0]: https://spacy.io/usage/v3
[usage guides]: https://spacy.io/usage/
[api reference]: https://spacy.io/api/
[gpu processing]: https://spacy.io/usage#gpu
[models]: https://spacy.io/models
[large language models]: https://spacy.io/usage/large-language-models
[universe]: https://spacy.io/universe
[spacy vs code extension]: https://github.com/explosion/spacy-vscode
[videos]: https://www.youtube.com/c/ExplosionAI
[online course]: https://course.spacy.io

> [blog]: https://explosion.ai/blog

Reviewer: ```suggestion
[blog]: https://explosion.ai
```
